### PR TITLE
Che/CRW branding+fixes+newAttribute

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

* Adding context info
* Fixing RAM/CPU resources table
* Doing a mockup for two images (removing Che-leaving just Workspace/Controler) - As PM suggested
* Adding new attribute for Keycloak for a proper synergy with the downstream docs

### CODE REVIEW

**Comment by nickboldt** [comment]
new images look great!

File: src/main/pages/che-7/administration-guide/con_che-architectural-components.adoc [hunk 1]
```diff
@@ -6,8 +6,10 @@
 As illustrated in the link:{site-baseurl}che-7/high-level-che-architecture[High-level {prod-short} architecture] article, {prod} components are:
 
 * A central workspace controller: an always running service that manages users workspaces
-* Users workspaces: container-based IDEs that the controller stops when the user stops coding
+* Users workspaces: container-based IDEs that the controller stops when the user stops coding.
 
+ifeval::["{project-context}" == "che"]
```
Comment (nickboldt) [inline src/main/pages/che-7/administration-guide/con_che-architectural-components.adoc]: now that you have a whitelabel version of the che-high-level image, you can probably remove this if-constraint.

File: src/main/pages/che-7/administration-guide/con_controller-requirements.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ The Workspace Controller consists of a set of five services running in five dist
 [cols="2,2,1,1", options="header"]
 .ControllerServices
 |===
-|Pod
+|{prod-short} Server and Dashboard
```
Comment (nickboldt) [inline src/main/pages/che-7/administration-guide/con_controller-requirements.adoc]: Uh.... no. It's still just a pod here. My suggestion was for line 16 below:

-|{prod-short}
+|{prod-short} Server and Dashboard

**Review by nickboldt — DISMISSED** [review]

**Comment by yhontyk** [comment]
@MichalMaler does it need backporting to 7.9.x? If yes, please cherry-pick. 
Otherwise, lgtm.

**Review by yhontyk — APPROVED** [review]

**Pull request merged**